<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::EditorFieldsOptionPage</class>
 <widget class="QWidget" name="QtGui::EditorFieldsOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fields</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="toBeShownLabel">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Fields to be shown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="fieldsListView">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
Remarks
&lt;ul style=&quot;margin-left: -20px; margin-top: -3px; text-indent: 0px &quot;&gt;
&lt;li&gt;Not all tag/container formats support all the fields selectable above.&lt;/li&gt;
&lt;li&gt;All unchecked fields are considered as unsupported fields (see Tag processing â†’ Handling of unsupported fields).&lt;/li&gt;
&lt;li&gt;Fields can be reordered via drag &amp;amp; drop.&lt;/li&gt;
&lt;li&gt;Changings take effect when opening the next file.&lt;/li&gt;
&lt;ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
